README.md

```diff
@@ -21,6 +21,153 @@ as these things will be provided by your cloud provider.
 
 But in case of a local deployment, navigate to the next section.
 
+#### Setting up remote cluster
+
+##### Setting up custom error backend
+
+On remote environment each namespace should have a custom error backend setup for the nginx. 
+This is done by creating the following ConfigMaps and Services in the namespace:
+
+This only needs to be applied once per namespace.
+
+Configuration for the default backend: service and handling of a 404 error on a subdomain level (wild card domain).
+
+Create `custom-error-backent.yaml` file with the following content. 
```

Comment:
1. Typo: `-backent`
2. Totally up to you, but personally I would prefer to have `custom-error-backend.yaml` as a file, not just in the README. It's so long (and important), it deserves to be a file I think.